src/OrbitGl/Track.cpp

```diff
@@ -12,7 +12,6 @@
 #include "CoreMath.h"
 #include "Geometry.h"
 #include "GlCanvas.h"
-#include "OrbitBase/ThreadUtils.h"
```

Comment:
Nit: This is an unrelated change. We try to keep PRs as focussed as possible which means this ideally shouldn't be in this PR.

Reply:
> This looks good to me.
> 
> More of a question to @karupayun: Can this be unit tested (easily)?

This function is a perfect candidate for `TEST(PrimitiveAssembler, ComplexShapes)` in `PrimitiveAssemblerTest.cpp`.

Reply:
Understood. I'll leave it this time, but I'll make sure to separate PRs next time.